src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1013,12 +1025,16 @@ def _start_manager_list(self, name, manager_list):
             name (str): manager name
             manager_list (list): list of SubprocessManager objects to start
         """
+        self.log.debug(
+            "## test.py - _start_manager_list 1: List = {}".format(
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions